### DISCUSSION TITLE
What level of DBAPI thread-safe is psycopg3?

### DISCUSSION THREAD

**Discussion by jalaziz** [psycopg/psycopg#514] [Q&A]
`psycopg2` had a [doc page](https://www.psycopg.org/docs/usage.html#thread-and-process-safety) detailing the thread-safety properties it had.

There's no equivalent doc for `psycopg3` as far as I can tell. While I'm sure `psycopg3` is thread-safe, it would be great to know what, if anything, is different from `psycopg2`?

**Comment by dvarrazzo** [comment]
Ok, will add. Spoiler: there should be no relevant difference compared with psycopg2, except for greenlet support which is untested.
